packages/graphics/graphics.5.1.1/opam

```diff
@@ -22,6 +22,7 @@ depends: [
   "conf-pkg-config" {os != "win32"}
   "ocaml" {>= "4.09.0~~"}
 ]
+conflicts: ["ocamlfind-secondary"]
```

Comment:
same question here. Something tells me the constraint should be in ocamlfind-secondary.